README.md

```diff
@@ -8,21 +8,32 @@ Please refer to [AppInsights-Home](https://github.com/Microsoft/ApplicationInsig
 
 Prerequisites
 -------------
-1. Java SDK 1.6 or higher
+1.  Java SDK 1.6 or higher
 
 Getting started
 ---------------
-1. Set JAVA_HOME environment variable to point to point to the [JDK installation directory]
-2. Recommended IDE is IntelliJ idea by using ‘Import Project’ rather than ‘Open Project’.
-3. You can use gradlew.bat on Windows to get gradle installed.
+1.  Set JAVA_HOME environment variable to point to point to the [JDK installation directory]
+2.  Recommended IDE is IntelliJ idea by using ‘Import Project’ rather than ‘Open Project’.
+3.  You can use gradlew.bat on Windows to get gradle installed.
 
 Using Eclipse IDE
 -----------------
-1. Install gradle from http://www.gradle.org/installation
-2. Add GRADLE_HOME/bin to your PATH environment variable
-3. In build.gradle add line [apply plugin: "eclipse"]
-4. In Eclipse used File->Import Existing Project in a workspace.
-5. Use [gradle build] to build the project from the command line.
+1.  Install gradle from http://www.gradle.org/installation
+2.  Add GRADLE_HOME/bin to your PATH environment variable
+3.  In build.gradle add line [apply plugin: "eclipse"]
+4.  In Eclipse used File->Import Existing Project in a workspace.
+5.  Use [gradle build] to build the project from the command line.
+
+CollectD Plugin - Optional
```

Comment:
This section is not aligned (use the View buttons to see how this file looks like in HTML).
Also as this item is optional consider placing it as a note (in a similar way to the instructions of how to build with Java 6)